### ISSUE TITLE
Assignable Profile has no way to naturally capture someone assigning something to someone

### ISSUE THREAD

**Issue by ViktorHaag** [1EdTech/caliper-spec#475]
You'd expect to have "[Instructor] Assigned [DigitalResource] to [Student]" but we can't do that in the profile. This seems an obvious gap.

If we have a use-case or recipe for capturing Assigning/Unassigning something ("activating" and "deactivating"?) then we should be explicit about that in the spec, I think?

**Comment by joshuamcghee** [comment]
@ViktorHaag I agree that this makes sense but reading the profile doc again it looks like that use case isn't even considered.  To me it seems that the profile as designed is more interested in looking at what has already been assigned/reassigned.

I would think that maybe we move forward as-is now and consider this as a future enhancement post 1.2.  Thoughts?

**Comment by ViktorHaag** [comment]
That's fine.
